jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/error/ExecutionErrorHandlerInterceptor.java

```diff
@@ -43,13 +43,19 @@ public final RequestContext execute( Executable executable, RequestContext ctx )
         if (counter == null) {
             counter = new AtomicInteger( 0 );
             invocationsCounter.set( counter );
+            if (executionErrorManager != null) {
```

Comment:
Sorry this is confusing to me - check for executionerrormanager being null is only relevant here if counter is also null? Same below in the finally method.

Reply:
yes, as this should only be done once for given transaction. interceptor can be many times on the stack as it is used by ksession and task service so we don't want it to be called many times.